packages/internal/src/generate/templates/mirror-cell.d.ts.template

```diff
@@ -10,6 +10,6 @@ export * from './${name}'
 type CellInputs = Omit<
   ComponentProps<SuccessType>,
   keyof QueryOperationResult | keyof ${queryResultType} | 'updating'
-> & ${queryVariablesType}
+> & (${queryVariablesType} extends Exact<{ [key: string]: never }> ? unknown : ${queryVariablesType})
```

Comment:
What I was thinking also - but two things:
a) Do we need to check for Exact ?
b) I don't think `unknown` is correct, it should be `{}` or some sort of empty type

Reply:
c) Lets move this into a utility type